### PR TITLE
Add option to disable marks, disable marks in `lowest` graphics profile

### PR DESCRIPTION
**Pull request by illwieckz**
- Add option to disable marks
- Disable marks in `lowest` graphics profile

Marks are very costly:

- https://github.com/Unvanquished/Unvanquished/issues/2663

Now I'm looking for a proper title for part where both particles and marks can be tweaked.

### CODE REVIEW

**Comment by slipher** [comment]
Are they really that costly if you have an actually bad GPU? Slow IPC operations can be a bottleneck if you have a fast GPU but with a slow GPU maybe rendering actually takes up most of the time and the IPC is not much slower on a weaker system.

**Comment by illwieckz** [comment]
If you have a bad GPU the models may be animated on CPU instead of GPU so anything saving CPU time is welcome.

I only disabled that in `lowest`, lowest already has things like the disablement of sky, etc. That's the very last resort.

Those traps done for setting up marks are very costly in CPU time anyway, so I may disable marks myself even if I enable the ultra preset. If you mix marks + sky + firebomb spam the game can be very laggy (typical dretchstorm game). I have a good Ryzen and the game can jump from 800fps to 100 just by spawning some firebombs, even after having optimized the particles, so I would not want to be one using an entry level laptop.

File: pkg/unvanquished_src.dpkdir/ui/options_graphics.rml [hunk 1]
```diff
@@ -226,6 +226,11 @@
 				</row>
 
 				<h2><translate>Particles</translate></h2>
+				<row>
+					<h3><translate>Marks</translate></h3>
+					<input cvar="cg_marks" type="checkbox" />
+					<p><translate>Allow marks to be painted on surfaces.</translate></p>
```
Comment (slipher) [inline pkg/unvanquished_src.dpkdir/ui/options_graphics.rml:232]: How about examples to make this clearer. `Draw marks (such as bullet holes or creep) on surfaces`

**Review by slipher — COMMENTED** [review]
Looks OK to me but it's blocked by #2682 aparently

File: pkg/unvanquished_src.dpkdir/ui/options_graphics.rml (hunk 1 above)
Comment (illwieckz) [reply pkg/unvanquished_src.dpkdir/ui/options_graphics.rml]: Done.

**Pull request merged**